examples/Mediawiki/QueryAPI.yaml

```diff
@@ -0,0 +1,78 @@
+suite: MediaWiki Query API
+description: Test fetching data from and about wikipages
+type: RPC
+
+setup:
+  # Create a new page
+  - request:
+      method: post
+      path: api.php
+      parameters:
+        action: edit
+        title: Query API
+        createonly: true
+        format: json
+      form-data:
+        summary: Create query api page
+        text: "The main page can be found at [[Main Page]]. There is also a great template at {{Hello}} and a  great cateogry  to review is: [[Category:Wiki]]"
+        token: +\
+    response:
+      body:
+        edit:
+          result: Success
+
+tests:
+  - description: Fetch links of a page
+    interaction:
+      - request:
+          path: api.php
+          parameters:
+            action: query
+            prop: links|categories|templates
+            titles: Query API
+            format: json
+        response:
```

Comment:
Haven't tried, but I think yaml supports the entire path in a single line, as in:
    response: body: query: pages:

Might be more readyble

Reply:
Will try

Reply:
Tried it with one of the responses and symfony's yaml parser throughs an error ` A colon cannot be used in an unquoted mapping value at line 269 (  
  near "response: body: edit: result: Success").`

Reply:
Meh. This would have been so much nicer ;)

Reply:
I agree